addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/channels.xml

```diff
@@ -5,6 +5,18 @@
 
 	<!-- Channel definitions -->
 
+    <channel-type id="rawmessage">
```

Comment:
It looks like this formatted a little bit different then the rest of the file. Did you format it with the ESH profile?